examples/acados_python/pendulum_example/export_pendulum_ode_model.py

```diff
@@ -63,10 +63,10 @@ def export_pendulum_ode_model():
     xdot = vertcat(x1_dot, theta_dot, v1_dot, dtheta_dot)
 
     # algebraic variables
-    z = SX.sym('z', 0)
+    z = None
```

Comment:
I lost track of the status here. Why can't `None` and `[]` be set in the constructor?

Reply:
actually both `None` and `[]` work now. Which is meant to be demonstrated in this file.